classes/salesforce.php

```diff
@@ -37,12 +37,14 @@ class Object_Sync_Sf_Salesforce {
 	*   Object for doing things to WordPress - retrieving data, cache, etc.
 	* @param string $slug
 	*   Slug for this plugin. Can be used for file including, especially
+	* @param string $option_prefix
+	*   Option prefix for this plugin. Used for getting and setting options, actions, etc.
 	* @param object $logging
 	*   Logging object for this plugin.
 	* @param array $schedulable_classes
 	*   array of classes that can have scheduled tasks specific to them
 	*/
-	public function __construct( $consumer_key, $consumer_secret, $login_url, $callback_url, $authorize_path, $token_path, $rest_api_version, $wordpress, $slug, $logging, $schedulable_classes ) {
+	public function __construct( $consumer_key, $consumer_secret, $login_url, $callback_url, $authorize_path, $token_path, $rest_api_version, $wordpress, $slug, $option_prefix, $logging, $schedulable_classes ) {
```

Comment:
Since `__construct` is a public method and this is the 1.4 release (not a 2.0 backwards compatibility breaking release), the new parameters should be optional. We can't guarantee developers haven't invoked this method.